docs/xls-65d-single-asset-vault/reference/transactions/vault-create.md

```diff
@@ -0,0 +1,90 @@
+---
+seo:
+    description: Creates a new vault object in the ledger.
+labels:
+  - Transactions
+  - Single Asset Vault
+---
+
+# VaultCreate
+
+[[Source]](https://github.com/XRPLF/rippled/blob/9d619b9dc579c592f0560c1b40fd3c98d7587d23/src/xrpld/app/tx/detail/VaultCreate.cpp "Source")
+
+Creates a new `Vault` object, an `MPTokenIssuance` ledger entry for the vault’s shares, and an `AccountRoot` for the vault’s [pseudo-account](https://github.com/XRPLF/XRPL-Standards/discussions/191).
+
+Only the Vault Owner can initiate this transaction.
+
+{% admonition type="info" name="Note" %}
+Currently, the same account that creates the vault must also create other protocols, though this may change in future.
+{% /admonition %}
+
```

Comment:
```suggestion
_(Requires the [Single Asset Vault amendment][] {% not-enabled /%})_
```
Just adding this in here to match the guidelines for easier doc migration later. This needs to be applied to all single asset vault pages. You'll also have to update the common-links.md file.